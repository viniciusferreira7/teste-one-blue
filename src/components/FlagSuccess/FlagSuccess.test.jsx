import { render, screen } from '@testing-library/react';
import { FlagSuccess } from '.';

describe('<FlagSuccess/>', () => {
  it('should render text', () => {
    render(<FlagSuccess>Sucesso</FlagSuccess>);
    expect(screen.getByText('Sucesso')).toBeInTheDocument();
  });
  it('should render component with styles', () => {
    render(<FlagSuccess>Sucesso</FlagSuccess>);
    expect(screen.getByText('Sucesso')).toHaveStyle(
      'color:rgba(93, 12, 255, 1)',
    );
  });

  it('should match snapshot', () => {
    render(<FlagSuccess>Sucesso</FlagSuccess>);
    expect(screen.getByText('Sucesso')).toMatchSnapshot();
  });
});
