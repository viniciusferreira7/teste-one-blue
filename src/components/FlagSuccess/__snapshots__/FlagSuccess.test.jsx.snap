// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlagSuccess/> should match snapshot 1`] = `
<p
  class="sc-bczRLJ gypqCX"
>
  Sucesso
</p>
`;
